tests/integration/ink-contracts/README.md

```diff
@@ -0,0 +1,8 @@
+## Ink! Contracts
+
+This directory contains Ink! contracts which are used in integration tests.
+
+`.wasm` files in this directory are used by `pallet-contracts` scenarios, typically `src/xvm.rs`. The json
+files are for informational purposes only and are not consumed by the tests.
+
+The source code for the contracts can be found at https://github.com/AstarNetwork/ink-test-contracts
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->